tools/hsettp/kvs-cn-tree-get.c

```diff
@@ -324,32 +326,31 @@ kvs_cn_tree_get_parse_values(cJSON *const body, int *const len, char ***const va
                     return merr(ENOMEM);
                 offset += 1;
 
-                entry = cJSON_GetObjectItemCaseSensitive(k, "job");
-                assert(cJSON_IsNull(entry) || cJSON_IsObject(entry));
-                if (cJSON_IsObject(entry)) {
+                job = cJSON_GetObjectItemCaseSensitive(k, "job");
+                if (cJSON_IsObject(job)) {
                     uint id;
                     ulong time;
                     const char *rule;
                     const char *wmesg;
                     const char *action;
 
-                    entry = cJSON_GetObjectItemCaseSensitive(k, "id");
```

Comment:
It's suprising this seemed to work with normal built.  But it failed miserably with LTO build.